04LessonFour/in-class-project/css/style.css

```diff
@@ -21,5 +21,13 @@ h1 {
 
```

Comment:
using what you know now about floats, try going back to this project to improve it! the spacing and alignment of your elements are still off.